.gitignore

```diff
@@ -30,3 +31,9 @@ db.sqlite3
 
 # dotenv (environment variables, used for secret/local stuff)
 .env
+
+#Name of the virtual environment
+my_project
+
+#Pipfile
+Pipfile
```

Comment:
per my comment, rather than this being ignored, I think requirements.txt should be replaced with Pipfile

Reply:
So should I just update the file name of `requirements.txt` as `Pipfile` ?

Reply:
No, you should keep the `Pipfile` as created by `pipenv --three install -r requirements.txt` and commit to the repository too, along with `Pipfile.lock` (and remove both from the `.gitignore`)

Reply:
However the `requirements.txt` contains one additional dependency other than those in the generated `Pipfile` named `six==1.11.0`. How to tackle this?

Reply:
Can you have a look into your `Pipfile.lock`? `six` should turn up there 😃